crates/wasm-metadata/Cargo.toml

```diff
@@ -13,6 +13,7 @@ workspace = true
 [dependencies]
 clap = { workspace = true, optional = true }
 anyhow = { workspace = true }
+comfy-table = { workspace = true }
```

Comment:
Could this perhaps be deferred to the CLI? I could imagine a `--format json` flag eventually being added to the CLI so having this as a CLI-only dependency I think might be better to have

Reply:
The CLI already has a working `--json` flag, here is what the output looks like for this branch:
<img width="580" alt="Screenshot 2024-12-10 at 17 53 43" src="https://github.com/user-attachments/assets/94f90af3-fdb7-4271-b355-0301cb135637">

That's not to say I'm not happy to move it though - would you still prefer we move it?

Reply:
Oh hey nice! Personally though I still would prefer to move this to the CLI as that's sort of where "rendering" happens and this is a render-only dependency which someone programmatically using `wasm-metadata` might not need.